soh/soh/SaveManager.cpp

```diff
@@ -58,7 +58,13 @@ void SaveManager::LoadRandomizerVersion1() {
     if(!CVar_GetS32("gRandomizer", 0)) return;
 
     for (int i = 0; i < ARRAY_COUNT(gSaveContext.itemLocations); i++) {
-        SaveManager::Instance->LoadData("get" + std::to_string(i), gSaveContext.itemLocations[i].get);
+        SaveManager::Instance->LoadStruct("get" + std::to_string(i), [&]() {
+            SaveManager::Instance->LoadData("rgID", gSaveContext.itemLocations[i].get.rgID);
+            SaveManager::Instance->LoadData("fakeRgID", gSaveContext.itemLocations[i].get.fakeRgID);
+            std::string trickName;
+            SaveManager::Instance->LoadData("trickName", trickName);
+            memcpy(gSaveContext.itemLocations[i].get.trickName, trickName.c_str(), trickName.length());
```

Comment:
another place where it'd be good to do something safer than a memcpy

Reply:
Hmmm, worth noting that Proxy had memcpy in his save file format update unless he changed it, I specifically followed that pattern. Also worth noting is that this is memcpying to an array of chars rather than a char*. Not sure if that makes a difference.